impl/src/main/java/dao/AnswerDao.java

```diff
@@ -106,30 +110,14 @@
             "AND answer.user_id=:userId")
     Observable<Void> updateAnswer(long userId, long answerId, Answer answer);
 
-    static final String S = "SELECT " +
-        "answer.id, " +
-        "answer.user_id, " +
-        "answer.answer, " +
-        "answer.created_at, " +
-        "answer.accepted, " +
-        "\"user\".picture, " +
-        "answer.slack_id, " +
-        "answer.question_id, " +
-        "\"user\".name AS created_by, " +
-        "question.user_id AS \"question.user_id\", " +
-        "(SELECT COALESCE(SUM(answer_vote.value), 0) FROM answer_vote WHERE answer_vote.answer_id = answer.id) AS votes " +
-        "FROM answer " +
-        "INNER JOIN \"user\" on \"user\".id = answer.user_id " +
-        "INNER JOIN question on question.id = answer.question_id " +
-        "WHERE answer.id=:id";
-
     @Query(
         "SELECT " +
             "answer.id, " +
             "answer.user_id, " +
             "answer.answer, " +
             "answer.created_at, " +
             "answer.accepted, " +
+            "answer.accepted_at, " +
```

Comment:
The "accepted" column becomes redundant with the addition of accepted_at. Perhaps remove it?

Reply:
Yes! Good point!